form.php

```diff
@@ -0,0 +1,46 @@
+<?php
+require_once 'functions.php';
+
+?>
+
+<!--write the form with inputs-->
+<html>
+<head>
+    <link rel="shortcut icon" href="images/favicon.ico"/>
+    <title>Add to the Cheese Collection</title>
+    <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link href="https://fonts.googleapis.com/css?family=Indie+Flower&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+
+<body>
+    <h1>Add a cheese to the collection!</h1>
+    <form action="index.php">
+        <input type="text" name="name" placeholder="Cheese name" formmethod="POST">
+        <input type="text" name="country" placeholder="Country of origin" formmethod="POST">
+        <input type="text" name="wine" placeholder="Wine pairing" formmethod="POST">
+        <input type="text" name="fact" placeholder="Fun fact" formmethod="POST">
+        <input type="text" name="photo" placeholder="Upload a photo" formmethod="POST">
```

Comment:
it was suggested by Mark to avoid an img input option as making it work could be a potential stumbling block